swiftpkg/internal/build_decls.bzl

```diff
@@ -29,7 +29,10 @@ def _to_starlark_parts(decl, indent):
     for c in decl.comments:
         parts.append(scg.indent(indent, "{}\n".format(c)))
     parts.append(scg.indent(indent, "{}(\n".format(decl.kind)))
-    parts.extend(scg.new_attr("name", decl.name, indent + 1))
+
+    # Name won't be set for `package` declarations
+    if decl.name:
+        parts.extend(scg.new_attr("name", decl.name, indent + 1))
```

Comment:
Just to confirm. The case when a name won't be present is the package declaration. Is that correct?

Reply:
Correct.